### ISSUE TITLE
LNDg: CSRF verification failed. 

### ISSUE THREAD

**Issue by robert-winkler** [raspiblitz/raspiblitz#4451]
I installed LNDg from the RaspiBlitz Menu v1.10.0 (upgraded from V.1.9.0) 

After installation, the login page is accessible, but after entering lndg-admin:password the following error occurs: 

~~~
Forbidden (403)
CSRF verification failed. Request aborted.
More information is available with DEBUG=True.
~~~

I tried after removing the /app-data directory on the HDD, and reinstalling, but without success. 

Any idea?

**Comment by rootzoll** [comment]
see #4418 - LNDg will be deactivated for repair in next release.

**Comment by rootzoll** [comment]
Might put out a bounty for repair.

**Comment by rootzoll** [comment]
Please wait for release v1.11.0 ... if LNDg still does not work on that version please write here.

**Comment by rootzoll** [comment]
OK looks like we got lndg fixed as part of v1.11.0 - closing for final release.
